File: Patterns/Pattern 15.cpp
class Solution {
public:
    void pattern15(int n) {
         for(int i = 0; i<n; i++)
        {
            for(char ch = 'A'; ch<'A'+(n-i); ch++)
            {
                cout<<ch;
            }
            cout << endl;
        }

    }
};

File: Patterns/Pattern 15.py
# Pattern 15: Inverted Alphabet Triangle
# A B C D E
# A B C D
# A B C
# A B
# A

def pattern15(n):
    for i in range(n, 0, -1):
        print(" ".join(chr(ord('A') + j) for j in range(i)))

pattern15(5)
